### PR TITLE
Change “home” to “Home”

### PR DESCRIPTION
**Pull request by revelcw**

### CODE REVIEW

**Review by maniator — CHANGES_REQUESTED** [review]
Why?

(also why are any of the letters caps aside for the first one?)

**Comment by revelcw** [comment]
I'm doing it to be consistent.

**Comment by revelcw** [comment]
> (also why are any of the letters caps aside for the first one?)

Because for a title you put them in all caps to stand out, except for "at", "the", "of", etc.

**Review by matthew-gerstman — APPROVED** [review]

**Comment by revelcw** [comment]
Thanks @matthew-gerstman

**Comment by maniator** [comment]
Can always use css `text-transform: capitalize;` instead :-)

File: src/pages/home/Header.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default () => (
       <img alt="useReactNYC logo" src={logo} />
       <h1>Welcome to useReactNYC</h1>
       <p className="subhead">
-        The home of New York City’s Newest and Most Exciting React Community
+        The Home of New York City’s Newest and Most Exciting React Community
```
Comment (maniator) [inline src/pages/home/Header.tsx:10]: I feel like this should be sentence case...

/shrug

Comment (revelcw) [reply src/pages/home/Header.tsx:10]: Yeah, but it's a big title

**Pull request merged**